datacatalog/cloud-client/src/main/java/com/example/datacatalog/CreateFilesetEntry.java

```diff
@@ -1,3 +1,4 @@
+// [END datacatalog_create_entry_group_tag]
```

Comment:
This change looks like a mistake. It doesn't seem right that a file wout start with an end tag.

Reply:
yea made a mistake after running the java formatter